src/app/ngrx/reducers/router.serializer.ts

```diff
@@ -1,15 +1,15 @@
 import { ActivatedRouteSnapshot, RouterStateSnapshot } from '@angular/router';
 import { RouterStateSerializer } from '@ngrx/router-store';
```

Comment:
This is that router serializer that is connected via the `@ngrx/router-store`. Whenever routing occurs in the app, the router-store produces a ROUTER_NAVIGATION action. This serializer reads the route url string and params and transforms the RouterStateSnapshot to our stripped down custom RouterParams model.